src/utils/logging.ts

```diff
@@ -36,7 +36,7 @@ type IExtendedAmpltitudeEvents =
   | AmplitudeEvent<'table expand lukket', { tekst: string; data: string }>
   | AmplitudeEvent<'help text åpnet', { tekst: string }>
   | AmplitudeEvent<'help text lukket', { tekst: string }>
-  | AmplitudeEvent<'alert', { tekst: string }>
+  | AmplitudeEvent<'alert vist', { tekst: string; variant: string }>
```

Comment:
Jeg er usikker i hvilken grad disse typene er i bruk, men hvis du først er innom kan du sørge for at objektet er oppdatert til å gjenspeile alle eventsene som brukes?

Reply:
Hør gjerne hva @bjorngi synes, jeg ser i gut history at han har innført dem.
Jeg mener at de bør enten slettes eller oppdateres, men ikke noe halvveis.